scripts/newsexsystem.gd

```diff
@@ -1951,8 +1984,10 @@ func endencounter():
 		if i.person.unique in ['dog','horse']:
 			continue
 		i.person.lewdness = i.lewd
-		if i.orgasms > 0:
+		if i.orgasms > 0 && !i.person.race_display in ['Succubus']: #ralphC - added "&& i.person.race_display != "Succubus""
```

Comment:
These if and elif statements are redundant. Check the race_display using a new if inside the `orgasm > 0` section.

Reply:
I was planning to add to the arrays in the future, but who knows when, so ok, will do.

Reply:
Then swap the commented condition for the used one until you need it later.

Reply:
fixed both for next commit